src/models/validators/SpecialistTestsCommonSchemaSubmitted.ts

```diff
@@ -76,3 +106,35 @@ export const testResultsCommonSchemaSpecialistTestsSubmitted =
       .items(testTypesCommonSchemaSpecialistTestsSubmitted)
       .required(),
   });
+
+export const testResultsIVADefectCommonSchemaSpecialistTestsSubmitted =
```

Comment:
Would `ivaDefectsCommonSchemaSpecialistTestsSubmitted` or something similar be more appropriate here given the defect array common schema is called 'defectsCommonSchemaSpecialistTestsSubmitted'

Reply:
The names of all structured like that throughout, why should this be different?

Reply:
As mentioned above, the existing defects schema is named:

defectsCommonSchemaSpecialistTestsSubmitted

and not 

testResultsDefectsCommonSchemaSpecialistTestsSubmitted

Reply:
As of right now it's called testResultsCommonSchemaSpecialistTestsSubmitted?